### ISSUE TITLE
Screenreader: Poll results do not show any results (numbers)

### ISSUE THREAD

**Issue by tibroc** [bigbluebutton/bigbluebutton#22098]
**Describe the bug**
In screenreader the poll results do not show the numbers of the results.

**To Reproduce**
1. Use a screenreader
2. Make a poll
3. Read the poll results with a screenreader

**Expected behavior**
The poll results should be legible by a screenreader

**BBB version:**
BBB 3.0-rc.1 with livekit enabled
